examples/fastify-plugin.ts

```diff
@@ -14,7 +14,7 @@ const inputData = z.object({
 const server = () => {
   const fastify = Fastify().withTypeProvider<TypeBoxTypeProvider>();
 
-  fastify.register(
+  void fastify.register(
```

Comment:
?

Reply:
Normally, I use `void` to communicate that we are not using the return value of this function even though it returns something, however, in this instance we probably should `await` this instead!

I'll fix that 👍